vdirsyncer/storage/ldap.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+import logging
+import ldap3
+
+from .base import Item, Storage
+from .. import exceptions
+
+ldap_logger = logging.getLogger(__name__)
+
+
+class LDAPStorage(Storage):
+
+    __doc__ = '''
```

Comment:
When implementing upload and update, we have to make it very clear which properties are even parsed and which ones are discarded.